api/spec/v1.yaml

```diff
@@ -1036,6 +1036,30 @@ paths:
                 $ref: '#/components/schemas/TokenHolderList'
         <<: *common_error_responses
 
+  /{runtime}/evm_tokens/{address}/nfts:
```

Comment:
Can you add this to `e2e_regresion/run.sh` please?

Reply:
might have to do this at the end of the PR train, as this'll take more work to normalize the metadata_accessed field (it's a timestamp). should probably also cache network items for the regression test x_x

Reply:
ok, discussed today, there's a way to blank it out in jq. added that. no cache. passes locally